symphony-bdk-legacy/symphony-api-client-java/src/test/java/it/clients/symphony/api/UsersClientTest.java

```diff
@@ -2,96 +2,1029 @@
+      assertEquals("test_2", userInfo.getUsername());
+      assertEquals("location", userInfo.getLocation());
+
+      final List<Avatar> avatars = userInfo.getAvatars();
+      assertNotNull(avatars);
+      assertEquals(2, avatars.size());
+
+      int i=0;
+
+      String expectedSize = null;
+      String expectedUrl = null;
+
+      for(Avatar avatar : avatars){
+        i++;
+        assertNotNull(avatar);
+        if(i==1){
+          expectedSize = "original";
+          expectedUrl = "../avatars/static/150/default.png";
+        } else if(i==2){
+          expectedSize = "small";
+          expectedUrl = "../avatars/static/50/default.png";
+        }
+        assertEquals(expectedSize, avatar.getSize());
+        assertEquals(expectedUrl, avatar.getUrl());
+      }
+    } catch (NoContentException nce) {
+      fail();
+    } catch (SymClientException sce) {
+      fail();
+    }
+  }
+
+  @Test
+  public void getUserFromEmailError() throws IOException {
+      final StubMapping stubMapping = stubGet(PodConstants.GETUSERSV3,
+          readResourceContent("/response_content/user_client/get_user_error.json"));
+
+    try {
+
+      assertNotNull(usersClient);
+      assertNotNull(stubMapping);
+
+      final ResponseDefinition responseDefinition = stubMapping.getResponse();
+      assertNotNull(responseDefinition);
+
+      final String content = responseDefinition.getBody();
+      assertNotNull(content);
+      final JsonNode jsonNode = stringToJson(content);
+      assertNotNull(jsonNode);
+
+      final JsonNode jsonNodeErrors = jsonNode.get("errors");
+      assertNotNull(jsonNodeErrors);
+
+      assertEquals(2, jsonNodeErrors.size());
+
+      final JsonNode jsonNodeError1 = jsonNodeErrors.get(0);
+      assertNotNull(jsonNodeError1);
+      final JsonNode errorField1 = jsonNodeError1.get("error");
+      assertNotNull(errorField1);
+      final JsonNode email = jsonNodeError1.get("email");
+      assertEquals("invalid.format", errorField1.asText());
+      assertEquals("test_2@.symphony.com", email.asText());
+
+      final JsonNode jsonNodeError2 = jsonNodeErrors.get(1);
+      assertNotNull(jsonNodeError2);
+      final JsonNode errorField2 = jsonNodeError2.get("error");
+      assertNotNull(errorField2);
+      final JsonNode id = jsonNodeError2.get("id");
+      assertEquals("invalid.format", errorField2.asText());
+      assertEquals(654321, id.intValue());
+
+    } catch (SymClientException sce) {
+      fail();
+    } catch (NoContentException nce){
+      fail();
+    }
+  }
+
+  private JsonNode stringToJson(String content) throws IOException {
+    final ObjectMapper mapper = new ObjectMapper();
+    final JsonFactory factory = mapper.getFactory();
+    assertNotNull(factory);
+    final JsonParser jsonParser = factory.createParser(content);
+    assertNotNull(jsonParser);
+    final JsonNode jsonNode = mapper.readTree(jsonParser);
+    assertNotNull(jsonNode);
+    return jsonNode;
+  }
```

Comment:
You could simplify this method, as you don't necessary need to assert the Jackson API here: 
```java
  private static JsonNode stringToJson(String content) throws IOException {
    return new JsonMapper().readTree(content);
  }
```
Also note that this method can be `static`